ausonius-editions.csl

```diff
@@ -93,8 +93,14 @@
       <name and="text" name-as-sort-order="all" sort-separator=", " initialize-with=". " delimiter-precedes-last="never"/>
       <label prefix=", "/>
       <substitute>
-        <names variable="editorial-director"/>
-        <names variable="editor"/>
+        <names variable="editorial-director">
```

Comment:
All of this is not necessary. The label from the main "names" will be used for the subsitute name variables. Please revert these changes.